go/vt/orchestrator/app/cli.go

```diff
@@ -51,29 +51,10 @@ func (a stringSlice) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 func (a stringSlice) Less(i, j int) bool { return a[i] < a[j] }
 
 var commandSynonyms = map[string]string{
-	"stop-slave":                  "stop-replica",
-	"start-slave":                 "start-replica",
-	"restart-slave":               "restart-replica",
-	"reset-slave":                 "reset-replica",
-	"restart-slave-statements":    "restart-replica-statements",
-	"relocate-slaves":             "relocate-replicas",
-	"regroup-slaves":              "regroup-replicas",
-	"move-up-slaves":              "move-up-replicas",
-	"repoint-slaves":              "repoint-replicas",
-	"enslave-siblings":            "take-siblings",
-	"enslave-master":              "take-master",
-	"get-candidate-slave":         "get-candidate-replica",
-	"move-slaves-gtid":            "move-replicas-gtid",
-	"regroup-slaves-gtid":         "regroup-replicas-gtid",
-	"which-cluster-osc-slaves":    "which-cluster-osc-replicas",
-	"which-cluster-gh-ost-slaves": "which-cluster-gh-ost-replicas",
-	"which-slaves":                "which-replicas",
-	"detach-slave":                "detach-replica-master-host",
-	"detach-replica":              "detach-replica-master-host",
-	"detach-slave-master-host":    "detach-replica-master-host",
-	"reattach-slave":              "reattach-replica-master-host",
-	"reattach-replica":            "reattach-replica-master-host",
-	"reattach-slave-master-host":  "reattach-replica-master-host",
+	"enslave-siblings": "take-siblings",
+	"enslave-primary":  "take-primary",
```

Comment:
do we need these mappings, or can we remove the `enslave` versions altogether?

Reply:
those mappings were made for backwards-compatibility, which is a non-issue in vitess. These should be removed.

Reply:
Removed